assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -68,20 +69,21 @@ export default function PropertySelect( { label } ) {
 	);
 
 	const { selectProperty } = useDispatch( MODULES_ANALYTICS_4 );
+	const viewContext = useContext( ViewContextContext );
 
 	const onChange = useCallback(
 		( index, item ) => {
 			const newPropertyID = item.dataset.value;
 			if ( propertyID !== newPropertyID ) {
+				const action =
+					newPropertyID === PROPERTY_CREATE
+						? 'change_property_new'
+						: 'change_property';
 				selectProperty( newPropertyID );
-				trackEvent(
-					'analytics_setup',
-					'property_change',
-					newPropertyID
-				);
+				trackEvent( `${ viewContext }_analytics`, action, 'ua' );
```

Comment:
It should use `ga4` label.

Reply:
https://github.com/google/site-kit-wp/pull/4321/commits/92eb7601bc79ac399b6f6ed7b79f787ef6b3c071